FirebaseRemoteConfig/Sources/RCNConfigDBManager.h

```diff
@@ -110,6 +111,15 @@ typedef void (^RCNDBLoadCompletion)(BOOL success,
 /// Insert or update the data in Personalization config.
 - (BOOL)insertOrUpdatePersonalizationConfig:(NSDictionary *)metadata fromSource:(RCNDBSource)source;
 
+/// Insert rollout metadata in rollout table.
+/// @param key        The string as a key of rollout metadata belongs to, which indicates whether
```

Comment:
nit: "Key indicating whether rollout metadata is fetched or active.."

Reply:
Fixed.